near-sdk/src/store/mod.rs

```diff
@@ -52,7 +52,7 @@
 //! - [`UnorderedMap`]: Storage version of [`std::collections::HashMap`]. No ordering
 //!   guarantees.
 //!
-//! - [`TreeMap`](TreeMap) (`unstable`): Storage version of [`std::collections::BTreeMap`]. Ordered by key,
+//! - [`TreeMap`] (`unstable`): Storage version of [`std::collections::BTreeMap`]. Ordered by key,
```

Comment:
this need `--features unstable` suffix to cmd, otherwise it's referenced from doc, but not present in code. 
probably that 's the reason....

also the behaviour slightly changes with toolchain version
```bash
❯ : rustdoc --version
rustdoc 1.83.0 (90b35a623 2024-11-26)
```